src/Models/Ban.php

```diff
@@ -68,4 +71,57 @@ public function scopeNotExpired(Builder $query): void
         $query->where('expired_at', '>', Carbon::now()->format('Y-m-d H:i:s'))
             ->orWhereNull('expired_at');
     }
+
+    public function scopeMeta(Builder $query, string $name, $value): void
```

Comment:
`scopeWhereMeta()`

Reply:
Since this method use `whereJsonContains`, the `whereMeta` make sense to me. I already edited this and added doc and test for these feature.